src/component/Photo_Carasel.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import Home from './Home.jsx'
+
+
+class Photo_Carasel extends React.Component {
```

Comment:
Should probably rename this to PhotoCarousel. I don't think underscore is part of naming convention.

Reply:
Fixed the naming convention. The photos are just a place holder for now. Since we are not pushing to git what do  push for our place holder. Ill have text in front of my images